index.bs

```diff
@@ -795,20 +789,19 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     <li><dfn>Pan</dfn> is a numeric camera setting that controls the pan of the camera. The setting represents pan in arc seconds, which are 1/3600th of a degree. Values are in the range from -180\*3600 arc seconds to +180\*3600 arc seconds. Positive values pan the camera clockwise as viewed from above, and negative values pan the camera counter clockwise as viewed from above.
 
-    Any algorithm which uses a {{MediaTrackConstraintSet}} object and its {{MediaTrackConstraintSet/pan}} dictionary member which exists after a possible normalization MUST <a>request permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true, and, optionally, consider its {{DevicePermissionDescriptor/deviceId}} member set to any appropriate device's deviceId.
+    Pan is also a boolean constrainable property corresponding to the camera's ability to pan (which results in a proper non-degenerate numeric setting range) and to the user agent's decision to expose that ability.
+    This boolean constraint is satisfied by a <a>settings dictionary</a> if the constraint value matches the existence of the numeric pan setting in the settings dictionary.
```

Comment:
The _constrainable property_ is never a boolean, only the constraint. Once https://github.com/w3c/mediacapture-main/pull/766 is merged, we can say:
```suggestion
    Constraints on pan influence camera selection through {{fitness distance}} toward cameras with the ability to pan. To exert this influence without overwriting the current pan setting, pan may be constrained to true. Conversely, constraining it to false disfavors cameras with the ability to pan.
```